alicloud/resource_alicloud_gpdb_instance_test.go

```diff
@@ -108,7 +108,7 @@ func testSweepGpdbInstances(region string) error {
 	return nil
 }
 
-func TestAccAlicloudGpdbInstance_classic(t *testing.T) {
+func _TestAccAlicloudGpdbInstance_classic(t *testing.T) {
```

Comment:
If you want to skip the testcase, you can add a prefix Skip in the testcase name.